plugin-build/build.gradle.kts

```diff
@@ -134,3 +135,5 @@ tasks.withType<Test> {
         showStandardStreams = true
     }
 }
+
+tasks.register<io.sentry.android.gradle.internal.ASMifyTask>("asmify")
```

Comment:
Why do you do this?

Reply:
wdym? why am I adding a new task, or why am I registering it here?

Reply:
Sorry. I was wondering why you're registering a new task of the `ASMifyTask` type inside the `plugin-build` build file

Reply:
I thought it's the best place for this task, cause the `plugin-build` has ASM dependencies in the classpath, which we can utilise when asmifying some classes. AsmifyTask itself is a little nifty helper, that allows us to get the asmified code of any `.class` file.

Reply:
do we need this so the plugin could do its job or is it related to the samples or tests?

Reply:
figured in this thread https://github.com/getsentry/sentry-android-gradle-plugin/pull/190#discussion_r721616166